src/completedTasks.js

```diff
@@ -0,0 +1,11 @@
+import { getList, storeList } from './localStorage.js';
+
+export default function completed(e) {
+  const list = getList();
```

Comment:
- [ ] I would recommend using the proper ES6 syntax to fulfill the project requirement and also writing clean code as ES6 syntax covers more work in less code.